Answers.md

```diff
@@ -1 +1,17 @@
+1. Describe the biggest difference between .forEach & .map.
 
+.forEach opperate exactly like a for loop itterating through each element and modifying, comparing, or returning a value.
+
+.map takes and array and returns a brand new array.
+
+2. What is the difference between a function and a method?
+there isn't?
```

Comment:
how do I call a function? how do I call a method?

Reply:
ha! now I get it. a method is called using dot notation ex: .push() a function is called by passing in an argument (or parameter...I always get those two confused).